main.py

```diff
@@ -1,43 +1,87 @@
-from fastapi import FastAPI;
-from MVC.controllers import ControllerYards, ControllerBeds, ControllerPlants
+from fastapi import FastAPI
+
+from MVC.controllers import ControllerBeds
+from MVC.controllers import ControllerPlants
+from MVC.controllers import ControllerYards
+from MVC.models import BedRequestCreate
+from MVC.models import PlantRequestCreate
+from MVC.models import YardRequestCreate
 
 yards_controller = ControllerYards()
 beds_controller = ControllerBeds()
 plants_controller = ControllerPlants()
 app = FastAPI()
 
 
-@app.get("/")
+@app.get('/')
 async def root():
-    return {"message": "This is the best farmer assistence app ever!"}
+    return {
+        'message': 'This is the best farmer assistence app ever!',
+    }
+
+# Yard endpoints
+# GET
+
 
-@app.get('/yards')
-def get_all():
-    return yards_controller.get_all()
+@app.get('/yard')
+def get_all_yards():
+    yards = yards_controller.get_all()
+    return ({
+        'message': 'Listing all yards',
+        'yards': yards,
+    })
 
 
-@app.get('/beds')
-def get_all():
-    return beds_controller.get_all()
+# POST
+@app.post('/yard')
+def create_yard(request: list[YardRequestCreate]):
+    message = 'Yards couldn´t be created'
+    if yards_controller.create(request=request):
+        message = 'Yards created successfully'
+    return ({
+        'message': message,
+    })
 
 
-@app.get('/plants')
-def get_all():
-    return plants_controller.get_all()
+# Bed endpoints
+# GET
+@app.get('/bed')
+def get_all_beds():
+    beds = beds_controller.get_all()
+    return ({
+        'message': 'Listing all beds',
+        'beds': beds,
+    })
 
 
+# POST
+@app.post('/bed')
+def create_bed(request: list[BedRequestCreate]):
+    message = 'Beds couldn´t be created'
+    if beds_controller.create(request=request):
+        message = 'Beds created successfully'
+    return ({
+        'message': message,
+    })
 
-# @app.post('/user/')
-# def post_add_user(user:User):
-#     return controller.add_user(user) 
 
-# @app.delete('/user/')
-# def delete_user(username_request:UserRequestDelete):
-#     return controller.delete_user(username_request)
+# Plant endpoints
+# GET
+@app.get('/plant')
+def get_all_plants():
+    plants = plants_controller.get_all()
+    return {
+        'message': 'Listing all plants',
+        'plants': plants,
+    }
 
-# @app.post('/login/')
-# def login(credentials_request:UserRequestLogin):
-#     return controller.login(credentials_request)
 
-if __name__ == '__main__':
-    main()
\ No newline at end of file
+# POST
+@app.post('/plant')
+def create_plant(request: list[PlantRequestCreate]):
+    message = 'Plants couldn´t be created'
```

Comment:
no asignamos valores, que no sabemos si van a suceder--- y si lo hacemos, lo hacemos en positivo, porq el caso negativo deberia ir en una exeption

message = ''"
o
message = "success", y si, solo si- falla- entonces couldn't be created